ui/lib/server/widget/user.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+import 'package:harmony/server/server.dart';
```

Comment:
Here's a hidden circular dependency. Instead of importing from the ```server/server package```, create a ```server/models/user.dart``` and ```server/models/message_data.dart``` and move those classes there. Then import those files into the server_page and this page.